components/alertbanner/stories/template.js

```diff
@@ -0,0 +1,75 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import { Template as Divider } from '@spectrum-css/divider/stories/template.js';
+import { Template as Button } from '@spectrum-css/button/stories/template.js';
+import { Template as CloseButton } from "@spectrum-css/closebutton/stories/template.js";
+import { Template as Icon } from '@spectrum-css/icon/stories/template.js';
+
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "${rootClass}",
```

Comment:
```suggestion
  rootClass = "${rootClass}",
```

Does this work? I don't think it will b/c it's trying to use itself to set itself.